src/util.ts

```diff
@@ -26,3 +27,29 @@ export function caseInsensitiveEqual<T extends string>(a: T, b: T): boolean {
  * @param handle The username
  */
 export const toHandle = (handle: string): string => `@${handle}`;
+
+/** Converts a rank into its value inside the score configuration */
+export function rankToScore(rank: number, scores: FellowsScore): number {
+  switch (rank) {
+    case 1:
+      return scores.dan1;
+    case 2:
+      return scores.dan2;
+    case 3:
+      return scores.dan3;
+    case 4:
+      return scores.dan4;
+    case 5:
+      return scores.dan5;
+    case 6:
+      return scores.dan6;
+    case 7:
+      return scores.dan7;
+    case 8:
+      return scores.dan8;
+    case 9:
+      return scores.dan9;
```

Comment:
could be just `return scores['dan' + rank]` ?

Reply:
I'm afraid not unless I convert the object into a collection of keys which I think would make it unreadable.

I prefer to add more lines if they are easier to read than to complicate the code for the sake of reducing its size.

Reply:
<img width="237" alt="image" src="https://github.com/paritytech/review-bot/assets/1177472/e4cad58f-a7a5-48b3-970f-b4bd445bfdc2">

but i see your point